pkg/models/licensing.go

```diff
@@ -14,7 +14,7 @@ type Licensing interface {
 
 	StateInfo() string
 
-	EnabledFeatures() map[string]bool
+	ListFeatures() []FeatureFlag
```

Comment:
@DanCech @AgnesToulet 

This is the main difference from the previous version.  This will let us define flags without them being enabled by default